Bonsai/Bonsai.csproj

```diff
@@ -40,45 +38,84 @@
       <LastGenOutput>Resources.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
-  <Target Name="ILRepack" AfterTargets="Build">
+
+  <!-- Embed System.Resources.Extensions and its dependencies, see Bonsai.SystemResourcesExtensionsSupport -->
```

Comment:
Do we still need this even if we are installing it as a dependency later on? I guess you do because of possible resources in NuGet.Design. Curious to know how big that makes the bootstrapper now.

Reply:
As noted during our meeting, yes this unfortunately has to be part of the bootstrapper. It can't show the package installation dialog without it.

The compressed size of the `Bonsai.exe` increased 0.06 MB as a result of this PR. These backport assemblies are not very big. (`System.Memory.dll` is only 139 KB uncompressed.)